### PR TITLE
Add field grants for single issues

### PR DESCRIPTION
**Pull request by cmgiven**
connects #3717

### CODE REVIEW

File: app/models/issue.rb [hunk 1]
```diff
@@ -46,6 +46,10 @@ def remanded?
     disposition == :remanded
   end
 
+  def closed?
+    !!disposition
```
Comment (amprokop) [inline app/models/issue.rb]: So remanded issues are considered closed then? A new issue is created when a remand occurs?

Comment (cmgiven) [reply app/models/issue.rb]: That's a fair point. Maybe we rename this to `has_disposition?` to avoid folks reading more into it than is there?

Comment (cmgiven) [reply app/models/issue.rb]: Actually, given the conversation on the other issue, this entire check is unnecessary. If there's no disposition, the type of the event will be nil, and so the event will be dropped.

Comment (amprokop) [reply app/models/issue.rb]: cool, so maybe this can be removed then?

Comment (cmgiven) [reply app/models/issue.rb]: Yup. Check out 90d600a.

**Pull request merged**